bitnami/airflow/templates/triggerer/statefulset.yaml

```diff
@@ -158,19 +160,17 @@ spec:
                 - -ec
                 - |
                   export CONNECTION_CHECK_MAX_COUNT=0
-                  timeout {{ $livenessTimeout }} airflow jobs check --job-type TriggererJob --local {{- if not .Values.diagnosticMode.enabled }} 2>/dev/null {{- end }}
+                  timeout {{ $livenessTimeout }} airflow jobs check --job-type TriggererJob --local --limit 0 {{- if not .Values.diagnosticMode.enabled }} 2>/dev/null {{- end }}
           {{- end }}
           {{- if .Values.triggerer.customReadinessProbe }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.triggerer.customReadinessProbe "context" $) | trim | nindent 12 }}
           {{- else if .Values.triggerer.readinessProbe.enabled }}
-          {{- $readinessTimeout := sub (int .Values.triggerer.readinessProbe.timeoutSeconds) 1 }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.triggerer.readinessProbe "enabled") "context" $) | nindent 12 }}
             exec:
               command:
-                - /bin/bash
-                - -ec
-                - |
-                  timeout {{ $readinessTimeout }} airflow jobs check --job-type TriggererJob --local {{- if not .Values.diagnosticMode.enabled }} 2>/dev/null {{- end }}
+                - pgrep
+                - -f
+                - "airflow triggerer"
```

Comment:
If you're going to use different readiness/liveness probes (because of kubescape), I'd switch the order and use the "complex" one for readiness and the "simple" `pgrep` one for liveness.